Cargo.lock

```diff
@@ -684,6 +684,7 @@ dependencies = [
 name = "test_workloads"
```

Comment:
nit: in commit message, paragraph 2: "...the host will _call_ TvmCpuGetRegisterSet..."

Reply:
Done.